Assets/Plugins/Source/Editor/ConfigEditors/PlatformConfigEditor.cs

```diff
@@ -34,15 +34,12 @@ namespace PlayEveryWare.EpicOnlineServices.Editor
     /// <typeparam name="T">Intended to be a type accepted by the templated class ConfigHandler.</typeparam>
     public abstract class PlatformConfigEditor<T> : ConfigEditor<T>, IPlatformConfigEditor where T : PlatformConfig
     {
-        /// <summary>
-        /// The platform that this PlatformConfigEditor represents.
-        /// </summary>
-        protected PlatformManager.Platform Platform;
-
-        protected PlatformConfigEditor(PlatformManager.Platform platform) :
-            base(PlatformManager.GetFullName(platform))
+        protected PlatformConfigEditor()
```

Comment:
I'm surprised this isn't calling base anymore. I guess it's because this doesn't use any collapsible areas?

Reply:
Nope - it's because calling the base is implied - it only needs to be explicitly done when there is a parameter to pass.